src/main/java/me/ajaja/module/footprint/domain/FootprintFactory.java

```diff
@@ -18,10 +19,11 @@ public static FreeFootprint createFreeFootprint(FootprintParam.Create create, St
 		);
 	}
 
-	public static KptFootprint createkptFootprint(
+	public static KptFootprint createKptFootprint(
```

Comment:
네이밍이 직관적인데 더 직관적인 네이밍도 괜찮을 것 같아요!
```suggestion
	public static KptFootprint kptTemplate(
```

Reply:
피드백 반영 : 881a536265c342e0863634eba8b68bda295d604a

동사 목적어가 가장 직관적이라고 생각했는데 변경하고 보니 더 괜찮네요??
원래 관례같은 건가요???

Reply:
맞는 지는 모르겠지만 클래스 네임에서 주는 정보도 있다고 생각합니다!
그래서 이렇게 바뀌면 다른 인지 수준을 줄 수 있다고 생각하고등요 ㅎㅎ